test/cppsuite/test_harness/workload/random_generator.cxx

```diff
@@ -66,6 +70,26 @@ random_generator::generate_pseudo_random_string(std::size_t length)
 random_generator::random_generator()
 {
     _generator = std::mt19937(std::random_device{}());
-    _distribution = std::uniform_int_distribution<>(0, _characters.size() - 1);
+    _alphanum_distrib = std::uniform_int_distribution<>(0, _pseudo_alphanum.size() - 1);
+    _alpha_distrib = std::uniform_int_distribution<>(0, _alphabet.size() - 1);
 }
+
+std::uniform_int_distribution<> &
+random_generator::get_distribution(characters_type type)
+{
+    if (type == characters_type::ALPHABET)
```

Comment:
With enums it's safer to have switch/case, otherwise, it's easy to miss a spot when a new character type is added.
The same should be done in `random_generator::generate_string` and `random_generator::get_characters`.